join_request/ggwpez.yml

```
@@ -0,0 +1,33 @@
+collective: Potoc
+member:
+  name: Oliver Tale-Yazdi
+  address: 16a357f5Sxab3V2ne4emGQvqJaCLeYpTMx3TCjnQhmJQ71DX
+  github: ggwpez
+  rank: 0
```

please fill your matrix handle as per https://github.com/super-collective/collective-cli/pull/4#discussion_r1586380064
```suggestion
  github: ggwpez
  matrix: 
  rank: 0
```